uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/my-leave/components/my-leave-calendar_test.js

```diff
@@ -1,23 +1,57 @@
 (function (CRM) {
   define([
     'common/angular',
-    'common/angularMocks',
+    'common/lodash',
+    'mocks/data/option-group-mock-data',
+    'mocks/data/public-holiday-data',
+    'mocks/data/work-pattern-data',
+    'mocks/data/leave-request-data',
+    'mocks/apis/absence-period-api-mock',
+    'mocks/apis/absence-type-api-mock',
+    'mocks/apis/leave-request-api-mock',
+    'mocks/apis/public-holiday-api-mock',
+    'mocks/apis/option-group-api-mock',
+    'mocks/apis/work-pattern-api-mock',
     'leave-absences/shared/config',
     'leave-absences/my-leave/app'
```

Comment:
I am testing `calendar` module only, but we need the `app` to be injected, otherwise we would need to inject all dependencies manually. We have done the same thing for other modules too.